pkg/state/state.go

```diff
@@ -3181,6 +3175,33 @@ func (st *HelmState) GenerateOutputDir(outputDir string, release *ReleaseSpec, o
 	return buf.String(), nil
 }
 
+func (st *HelmState) GenerateChartPath(chartName string, outputDir string, release *ReleaseSpec, outputDirTemplate string) (string, error) {
```

Comment:
A little comment to explain the function? And should we not expose the function, and remove the receiver *HelmState as well?

```suggestion
// generateChartPath ....
func generateChartPath(chartName string, outputDir string, release *ReleaseSpec, outputDirTemplate string) (string, error) {
```

Reply:
Good point!